sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/files/commonconfig-6.12

```diff
@@ -460,6 +459,7 @@ CONFIG_MMC_BLOCK_MINORS=16
 CONFIG_MMC_SDHCI=m
 CONFIG_MMC_SDHCI_PCI=m
 CONFIG_MODULES=y
+CONFIG_MODULE_COMPRESS=y
 CONFIG_MODULE_COMPRESS_XZ=y
```

Comment:
Hm, I'm wondering if it would make sense to use `CONFIG_MODULE_COMPRESS_ZSTD` instead. Probably not much, given that we already compress initrd with zstd. Not in this PR though.

Reply:
CONFIG_MODULE_COMPRESS was required in order to use the other configs, I did not want to change other configs in this PR.

Reply:
I tried zstd on 6.6 to see if it would help with our space issues. It actually makes things larger, but I now realise that's expected. zstd beats xz at speed, not space. If anything, we should compress the rest with xz.

Reply:
Huh, interesting. I thought that zstd could compress more (by not much, though), but that probably would need cranking up some compression level flag to the max value.

Reply:
Nope, even at the max, xz generally still wins.